e2e/test/BulkOperationsE2ETests.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Shared;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    [TestClass]
+    [TestCategory("IoTHub-E2E")]
+    public class BulkOperationsE2ETests : IDisposable
+    {
+        private readonly string DevicePrefix = $"E2E_{nameof(BulkOperationsE2ETests)}_";
+        private readonly string ModulePrefix = $"E2E_{nameof(BulkOperationsE2ETests)}_";
+
+        private static TestLogging _log;
+        private readonly ConsoleEventListener _listener;
+
+        public BulkOperationsE2ETests()
+        {
+            _listener = TestConfig.StartEventListener();
+            _log = TestLogging.GetInstance();
```

Comment:
Static initializers should never be in a non-static ctor. See other tests for reference.

Reply:
This isn't used - please remove everything that's not used.